src/apify_client.ts

```diff
@@ -312,8 +312,9 @@ export class ApifyClient {
     }
 
     async setStatusMessage(message: string, options?: SetStatusMessageOptions): Promise<void> {
-        const runId = process.env[ENV_VARS.ACTOR_RUN_ID];
-        if (!runId) { throw new Error(`Environment variable ${ENV_VARS.ACTOR_RUN_ID} is not set!`); }
+        // TODO: Keep backwards compatibility or just do a breaking change?
```

Comment:
I think we can make it breaking as it is used only on the platform, where we already fix it.